tutorial/markdown/java/spring-boot.md

```diff
@@ -53,11 +90,23 @@ mvn package
 
 ### Database Server Configuration
 
-All configuration for communication with the database is stored in the `/src/main/resources/application.properties` file.  This includes the connection string, username, and password.  The default username is assumed to be `Administrator` and the default password is assumed to be `password`.  If these are different in your environment you will need to change them before running the application.
+the `CouchbaseConfig` class is a Spring configuration class responsible for setting up the connection to a Couchbase database in a Spring Boot application. It defines two beans:
+
+`getCouchbaseCluster()`: This bean creates and configures a connection to the Couchbase cluster using the provided hostname, username, and password.
+
+`getCouchbaseBucket(Cluster cluster)`: This bean creates a Couchbase bucket within the cluster if it doesn't already exist and returns the Bucket object associated with the specified bucket name.
 
-### Dependency Injection via DBSetupRunner Class
+Additionally, the class includes optional code for enabling `TLS (Transport Layer Security)` to secure communication between the application and the Couchbase server if required.
```

Comment:
This should not be required anymore. By default, Capella should connect with TLS enabled.

Reply:
removed this unnecessary comment